job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext/buildResult.groovy

```diff
@@ -0,0 +1,25 @@
+job('example') {
+    triggers {
+        buildResult('H/10 * * * *') {
+            combinedJobs()
+            triggerInfo('My job 1') {
```

Comment:
Do we really need the nested closure? `triggerInfo(String job, BuildResult... buildResult)` would reduce the complexity.